src/linux/backend-v4l2.cpp

```diff
@@ -2255,14 +2345,34 @@ namespace librealsense
                 throw linux_backend_exception(_md_name + " ioctl(VIDIOC_G_FMT) for metadata node failed");
             }
 
-            if (fmt.type != LOCAL_V4L2_BUF_TYPE_META_CAPTURE)
+            if (fmt.type != _md_type)
                 throw linux_backend_exception("ioctl(VIDIOC_G_FMT): " + _md_name + " node is not metadata capture");
 
             bool success = false;
+
             for (const uint32_t& request : { V4L2_META_FMT_D4XX, V4L2_META_FMT_UVC})
             {
                 // Configure metadata format - try d4xx, then fallback to currently retrieve UVC default header of 12 bytes
-                memcpy(fmt.fmt.raw_data,&request,sizeof(request));
+                memcpy(fmt.fmt.raw_data, &request, sizeof(request));
+                // use only for IPU6?
+                if ((_dev.cap.capabilities & V4L2_CAP_VIDEO_CAPTURE_MPLANE) && !is_platform_jetson())
+                {
+                    /* Sakari patch for videodev2.h. This structure will be within kernel > 6.4 */
+                    struct v4l2_meta_format {
```

Comment:
Can we share the patch link instead of writing "Sakari patch"?

Reply:
I will append line for submitted kernel patch